packages/dd-trace/src/config.js

```diff
@@ -664,9 +664,9 @@ class Config {
     this._setBoolean(env, 'profiling.enabled', coalesce(DD_EXPERIMENTAL_PROFILING_ENABLED, DD_PROFILING_ENABLED))
     this._setString(env, 'profiling.exporters', DD_PROFILING_EXPORTERS)
     this._setBoolean(env, 'profiling.sourceMap', DD_PROFILING_SOURCE_MAP && !isFalse(DD_PROFILING_SOURCE_MAP))
-    if (DD_INJECTION_ENABLED) {
+    if (DD_PROFILING_ENABLED === 'auto' || DD_INJECTION_ENABLED) {
       this._setBoolean(env, 'profiling.ssi', true)
-      if (DD_INJECTION_ENABLED.split(',').includes('profiler')) {
+      if (DD_PROFILING_ENABLED === 'auto' || DD_INJECTION_ENABLED.split(',').includes('profiler')) {
```

Comment:
Is it `profiler` or `profiling` that must be present in `DD_PROFILING_ENABLED` ?
There is a `profiling` [here](https://github.com/DataDog/dd-trace-js/blob/7b2dee1b05a7f4381a9755f097232c9ee44df2b1/packages/dd-trace/src/config.js#L725).

Reply:
It is 'profiler'. 'profiling' was a bug on my part. I can quickly fix it in this PR.